tests/test_pvsystem.py

```diff
@@ -2181,6 +2181,58 @@ def test_pvwatts_dc_series():
     assert_series_equal(expected, out)
 
 
+def test_pvwatts_dc_scalars_with_k():
+    expected = 8.9125
+    out = pvsystem.pvwatts_dc(100, 30, 100, -0.003, 25, 0.01)
```

Comment:
generally preferable to call keyword arguments as keyword arguments and only supply the one that deviates from default. So in this case you'd have `pvsystem.pvwatts_dc(100, 30, 100, -0.003, k=0.01)`

Reply:
And similar changes below